docs/modules/integrate/pages/feast-config.adoc

```diff
@@ -0,0 +1,70 @@
+= Configure Feature Store
+:page-enterprise: true
+:description: To use a Feast project, you must configure the feature store. The configuration is defined in a YAML configuration file.
+
+{description}
+
```

Comment:
It would also be great to introduce the "feature repository" concept.
It's actually a directory that stores configuration and feature definitions.
The user is able to change the feature repository (by default the current dir) using the `-c DIRECTORY` flag.
https://docs.feast.dev/reference/feature-repository